### ISSUE TITLE
Testing would be great

### ISSUE THREAD

**Issue by DaveParr** [satRdays/satRday_site_template#22]
There is currently no automated testing for this project, though there really probably should be.

Any ideas welcome.

**Comment by aastha44ma** [comment]
Yes the more a product is tested, the more it can be improved and made better.
